### ISSUE TITLE
2025-05-20, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#175]
# Today's Arxiv Papers

(1) [Unlocking Non-Invasive Brain-to-Text](http://arxiv.org/abs/2505.13446v1), 2025-05-19

(2) [Trust, But Verify: A Self-Verification Approach to Reinforcement Learning with Verifiable Rewards](http://arxiv.org/abs/2505.13445v1), 2025-05-19

(3) [Optimizing Anytime Reasoning via Budget Relative Policy Optimization](http://arxiv.org/abs/2505.13438v1), 2025-05-19

(4) [SMOTExT: SMOTE meets Large Language Models](http://arxiv.org/abs/2505.13434v1), 2025-05-19

(5) [Fine-tuning Quantized Neural Networks with Zeroth-order Optimization](http://arxiv.org/abs/2505.13430v1), 2025-05-19

(6) [MM-PRM: Enhancing Multimodal Mathematical Reasoning with Scalable Step-Level Supervision](http://arxiv.org/abs/2505.13427v1), 2025-05-19

(7) [Learnware of Language Models: Specialized Small Language Models Can Do Big](http://arxiv.org/abs/2505.13425v1), 2025-05-19

(8) [Make Still Further Progress: Chain of Thoughts for Tabular Data Leaderboard](http://arxiv.org/abs/2505.13421v1), 2025-05-19

(9) [FEALLM: Advancing Facial Emotion Analysis in Multimodal Large Language Models with Emotional Synergy and Reasoning](http://arxiv.org/abs/2505.13419v1), 2025-05-19

(10) [Dementia Through Different Eyes: Explainable Modeling of Human and LLM Perceptions for Early Awareness](http://arxiv.org/abs/2505.13418v1), 2025-05-19

(11) [Gluon: Making Muon & Scion Great Again! (Bridging Theory and Practice of LMO-based Optimizers for LLMs)](http://arxiv.org/abs/2505.13416v1), 2025-05-19

(12) [CoT-Kinetics: A Theoretical Modeling Assessing LRM Reasoning Process](http://arxiv.org/abs/2505.13408v1), 2025-05-19

(13) [AutoMathKG: The automated mathematical knowledge graph based on LLM and vector database](http://arxiv.org/abs/2505.13406v1), 2025-05-19

(14) [MR. Judge: Multimodal Reasoner as a Judge](http://arxiv.org/abs/2505.13403v1), 2025-05-19

(15) [CompeteSMoE -- Statistically Guaranteed Mixture of Experts Training via Competition](http://arxiv.org/abs/2505.13380v1), 2025-05-19

(16) [Thinkless: LLM Learns When to Think](http://arxiv.org/abs/2505.13379v1), 2025-05-19

(17) [Seeing, Saying, Solving: An LLM-to-TL Framework for Cooperative Robots](http://arxiv.org/abs/2505.13376v1), 2025-05-19

(18) [What Prompts Don't Say: Understanding and Managing Underspecification in LLM Prompts](http://arxiv.org/abs/2505.13360v1), 2025-05-19

(19) [Multi-Armed Bandits Meet Large Language Models](http://arxiv.org/abs/2505.13355v1), 2025-05-19

(20) [Sense and Sensitivity: Examining the Influence of Semantic Recall on Long Context Code Reasoning](http://arxiv.org/abs/2505.13353v1), 2025-05-19

(21) [Investigating the Vulnerability of LLM-as-a-Judge Architectures to Prompt-Injection Attacks](http://arxiv.org/abs/2505.13348v1), 2025-05-19

(22) [J4R: Learning to Judge with Equivalent Initial State Group Relative Preference Optimization](http://arxiv.org/abs/2505.13346v1), 2025-05-19

(23) [Occult: Optimizing Collaborative Communication across Experts for Accelerated Parallel MoE Training and Inference](http://arxiv.org/abs/2505.13345v1), 2025-05-19

(24) [Contextual Paralinguistic Data Creation for Multi-Modal Speech-LLM: Data Condensation and Spoken QA Generation](http://arxiv.org/abs/2505.13338v1), 2025-05-19

(25) [Rethinking Stateful Tool Use in Multi-Turn Dialogues: Benchmarks and Challenges](http://arxiv.org/abs/2505.13328v1), 2025-05-19

(26) [Thinking Short and Right Over Thinking Long: Serving LLM Reasoning Efficiently and Accurately](http://arxiv.org/abs/2505.13326v1), 2025-05-19

(27) [VesselGPT: Autoregressive Modeling of Vascular Geometry](http://arxiv.org/abs/2505.13318v1), 2025-05-19

(28) [GUARD: Generation-time LLM Unlearning via Adaptive Restriction and Detection](http://arxiv.org/abs/2505.13312v1), 2025-05-19

(29) [Seek in the Dark: Reasoning via Test-Time Instance-Level Policy Gradient in Latent Space](http://arxiv.org/abs/2505.13308v1), 2025-05-19

(30) [RBF++: Quantifying and Optimizing Reasoning Boundaries across Measurable and Unmeasurable Capabilities for Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.13307v1), 2025-05-19

(31) [I'll believe it when I see it: Images increase misinformation sharing in Vision-Language Models](http://arxiv.org/abs/2505.13302v1), 2025-05-19

(32) [Cross-Cloud Data Privacy Protection: Optimizing Collaborative Mechanisms of AI Systems by Integrating Federated Learning and LLMs](http://arxiv.org/abs/2505.13292v1), 2025-05-19

(33) [TimeSeriesGym: A Scalable Benchmark for (Time Series) Machine Learning Engineering Agents](http://arxiv.org/abs/2505.13291v1), 2025-05-19

(34) [Hybrid Voting-Based Task Assignment in Modular Construction Scenarios](http://arxiv.org/abs/2505.13278v1), 2025-05-19

(35) [CSC-SQL: Corrective Self-Consistency in Text-to-SQL via Reinforcement Learning](http://arxiv.org/abs/2505.13271v1), 2025-05-19

(36) [Distilling a speech and music encoder with task arithmetic](http://arxiv.org/abs/2505.13270v1), 2025-05-19

(37) [Are requirements really all you need? A case study of LLM-driven configuration code generation for automotive simulations](http://arxiv.org/abs/2505.13263v1), 2025-05-19

(38) [From Automation to Autonomy: A Survey on Large Language Models in Scientific Discovery](http://arxiv.org/abs/2505.13259v1), 2025-05-19

(39) [Effective and Transparent RAG: Adaptive-Reward Reinforcement Learning for Decision Traceability](http://arxiv.org/abs/2505.13258v1), 2025-05-19

(40) [HeteroSpec: Leveraging Contextual Heterogeneity for Efficient Speculative Decoding](http://arxiv.org/abs/2505.13254v1), 2025-05-19

(41) [Natural Language Planning via Coding and Inference Scaling](http://arxiv.org/abs/2505.13252v1), 2025-05-19

(42) [RN-F: A Novel Approach for Mitigating Contaminated Data in Large Language Models](http://arxiv.org/abs/2505.13249v1), 2025-05-19

(43) [Agentic Publications: An LLM-Driven Framework for Interactive Scientific Publishing, Supplementing Traditional Papers with AI-Powered Knowledge Systems](http://arxiv.org/abs/2505.13246v1), 2025-05-19

(44) [JNLP at SemEval-2025 Task 11: Cross-Lingual Multi-Label Emotion Detection Using Generative Models](http://arxiv.org/abs/2505.13244v1), 2025-05-19

(45) [SAKURA: On the Multi-hop Reasoning of Large Audio-Language Models Based on Speech and Audio Information](http://arxiv.org/abs/2505.13237v1), 2025-05-19

(46) [From Local Details to Global Context: Advancing Vision-Language Models with Attention-Based Selection](http://arxiv.org/abs/2505.13233v1), 2025-05-19

(47) [SeedBench: A Multi-task Benchmark for Evaluating Large Language Models in Seed Science](http://arxiv.org/abs/2505.13220v1), 2025-05-19

(48) [Picturized and Recited with Dialects: A Multimodal Chinese Representation Framework for Sentiment Analysis of Classical Chinese Poetry](http://arxiv.org/abs/2505.13210v1), 2025-05-19

(49) [Quantum Knowledge Distillation for Large Language Models](http://arxiv.org/abs/2505.13205v1), 2025-05-19

(50) [Alignment-Augmented Speculative Decoding with Alignment Sampling and Conditional Verification](http://arxiv.org/abs/2505.13204v1), 2025-05-19

(51) [Adversarial Testing in LLMs: Insights into Decision-Making Vulnerabilities](http://arxiv.org/abs/2505.13195v1), 2025-05-19

(52) [True Zero-Shot Inference of Dynamical Systems Preserving Long-Term Statistics](http://arxiv.org/abs/2505.13192v1), 2025-05-19

(53) [ViPlan: A Benchmark for Visual Planning with Symbolic Predicates and Vision-Language Models](http://arxiv.org/abs/2505.13180v1), 2025-05-19

(54) [ToolSpectrum : Towards Personalized Tool Utilization for Large Language Models](http://arxiv.org/abs/2505.13176v1), 2025-05-19

(55) [Enhancing LLMs for Time Series Forecasting via Structure-Guided Cross-Modal Alignment](http://arxiv.org/abs/2505.13175v1), 2025-05-19

(56) [A Case Study of Cross-Lingual Zero-Shot Generalization for Classical Languages in LLMs](http://arxiv.org/abs/2505.13173v1), 2025-05-19

(57) [Positional Fragility in LLMs: How Offset Effects Reshape Our Understanding of Memorization Risks](http://arxiv.org/abs/2505.13171v1), 2025-05-19

(58) [Role-Playing Evaluation for Large Language Models](http://arxiv.org/abs/2505.13157v1), 2025-05-19

(59) [Tianyi: A Traditional Chinese Medicine all-rounder language model and its Real-World Clinical Practice](http://arxiv.org/abs/2505.13156v1), 2025-05-19

(60) [What if Deception Cannot be Detected? A Cross-Linguistic Study on the Limits of Deception Detection from Text](http://arxiv.org/abs/2505.13147v1), 2025-05-19

(61) [Auditing Meta-Cognitive Hallucinations in Reasoning Large Language Models](http://arxiv.org/abs/2505.13143v1), 2025-05-19

(62) [Understanding Cross-Lingual Inconsistency in Large Language Models](http://arxiv.org/abs/2505.13141v1), 2025-05-19

(63) [Optimizing Retrieval Augmented Generation for Object Constraint Language](http://arxiv.org/abs/2505.13129v1), 2025-05-19

(64) [Zero-Shot Iterative Formalization and Planning in Partially Observable Environments](http://arxiv.org/abs/2505.13126v1), 2025-05-19

(65) [Benchmarking and Confidence Evaluation of LALMs For Temporal Reasoning](http://arxiv.org/abs/2505.13115v1), 2025-05-19

(66) [Why Knowledge Distillation Works in Generative Models: A Minimal Working Explanation](http://arxiv.org/abs/2505.13111v1), 2025-05-19

(67) [FreeKV: Boosting KV Cache Retrieval for Efficient LLM Inference](http://arxiv.org/abs/2505.13109v1), 2025-05-19

(68) [Fixing 7,400 Bugs for 1$: Cheap Crash-Site Program Repair](http://arxiv.org/abs/2505.13103v1), 2025-05-19

(69) [LLM-KG-Bench 3.0: A Compass for SemanticTechnology Capabilities in the Ocean of LLMs](http://arxiv.org/abs/2505.13098v1), 2025-05-19

(70) [The Effect of Language Diversity When Fine-Tuning Large Language Models for Translation](http://arxiv.org/abs/2505.13090v1), 2025-05-19

(71) [Universal Semantic Disentangled Privacy-preserving Speech Representation Learning](http://arxiv.org/abs/2505.13085v1), 2025-05-19

(72) [MultiActor-Audiobook: Zero-Shot Audiobook Generation with Faces and Voices of Multiple Speakers](http://arxiv.org/abs/2505.13082v1), 2025-05-19

(73) [Walking the Tightrope: Disentangling Beneficial and Detrimental Drifts in Non-Stationary Custom-Tuning](http://arxiv.org/abs/2505.13081v1), 2025-05-19

(74) [Advancing Sequential Numerical Prediction in Autoregressive Models](http://arxiv.org/abs/2505.13077v1), 2025-05-19

(75) [The Hidden Dangers of Browsing AI Agents](http://arxiv.org/abs/2505.13076v1), 2025-05-19

(76) [Structure-Aware Corpus Construction and User-Perception-Aligned Metrics for Large-Language-Model Code Completion](http://arxiv.org/abs/2505.13073v1), 2025-05-19

(77) [Hearing from Silence: Reasoning Audio Descriptions from Silent Videos via Vision-Language Model](http://arxiv.org/abs/2505.13062v1), 2025-05-19

(78) [Automatic mixed precision for optimizing gained time with constrained loss mean-squared-error based on model partition to sequential sub-graphs](http://arxiv.org/abs/2505.13060v1), 2025-05-19

(79) [CAIM: Development and Evaluation of a Cognitive AI Memory Framework for Long-Term Interaction with Intelligent Agents](http://arxiv.org/abs/2505.13044v1), 2025-05-19

(80) [KIT's Offline Speech Translation and Instruction Following Submission for IWSLT 2025](http://arxiv.org/abs/2505.13036v1), 2025-05-19

(81) [MMAR: A Challenging Benchmark for Deep Reasoning in Speech, Audio, Music, and Their Mix](http://arxiv.org/abs/2505.13032v1), 2025-05-19

(82) [MindOmni: Unleashing Reasoning Generation in Vision Language Models with RGPO](http://arxiv.org/abs/2505.13031v1), 2025-05-19

(83) [Evaluatiing the efficacy of LLM Safety Solutions : The Palit Benchmark Dataset](http://arxiv.org/abs/2505.13028v1), 2025-05-19

(84) [Step-wise Adaptive Integration of Supervised Fine-tuning and Reinforcement Learning for Task-Specific LLMs](http://arxiv.org/abs/2505.13026v1), 2025-05-19

(85) [Adversarial Reasoning for Repair Based on Inferred Program Intent](http://arxiv.org/abs/2505.13008v1), 2025-05-19

(86) [EffiBench-X: A Multi-Language Benchmark for Measuring Efficiency of LLM-Generated Code](http://arxiv.org/abs/2505.13004v1), 2025-05-19

(87) [Fractured Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.12992v1), 2025-05-19

(88) [Personalized Fine-Tuning with Controllable Synthetic Speech from LLM-Generated Transcripts for Dysarthric Speech Recognition](http://arxiv.org/abs/2505.12991v1), 2025-05-19

(89) [An Empirical Study of Many-to-Many Summarization with Large Language Models](http://arxiv.org/abs/2505.12983v1), 2025-05-19

(90) [From Assistants to Adversaries: Exploring the Security Risks of Mobile LLM Agents](http://arxiv.org/abs/2505.12981v1), 2025-05-19

(91) [A Structured Literature Review on Traditional Approaches in Current Natural Language Processing](http://arxiv.org/abs/2505.12970v1), 2025-05-19

(92) [MA-COIR: Leveraging Semantic Search Index and Generative Models for Ontology-Driven Biomedical Concept Recognition](http://arxiv.org/abs/2505.12964v1), 2025-05-19

(93) [DGRO: Enhancing LLM Reasoning via Exploration-Exploitation Control and Reward Variance Management](http://arxiv.org/abs/2505.12951v1), 2025-05-19

(94) [GuRE:Generative Query REwriter for Legal Passage Retrieval](http://arxiv.org/abs/2505.12950v1), 2025-05-19

(95) [A3 : an Analytical Low-Rank Approximation Framework for Attention](http://arxiv.org/abs/2505.12942v1), 2025-05-19

(96) [Leveraging LLM Inconsistency to Boost Pass@k Performance](http://arxiv.org/abs/2505.12938v1), 2025-05-19

(97) [Do Not Let Low-Probability Tokens Over-Dominate in RL for LLMs](http://arxiv.org/abs/2505.12929v1), 2025-05-19

(98) [CPRet: A Dataset, Benchmark, and Model for Retrieval in Competitive Programming](http://arxiv.org/abs/2505.12925v1), 2025-05-19

(99) [The Traitors: Deception and Trust in Multi-Agent Language Model Simulations](http://arxiv.org/abs/2505.12923v1), 2025-05-19

(100) [Sinusoidal Initialization, Time for a New Start](http://arxiv.org/abs/2505.12909v1), 2025-05-19
